Makefile

```diff
@@ -73,42 +73,46 @@ $(subst .doc/models/template,,$(MODELS_D)): .install/models/template
+$(call depends,modules/priors): | .install/base/utils .install/base/logger
+$(call depends,modules/rtm): | .install/base/logger .install/modules/assim.batch
+$(call depends,modules/uncertainty): | .install/base/utils .install/modules/priors .install/base/db .install/modules/emulator .install/base/logger
+$(call depends,models/biocro): | .install/base/logger .install/base/remote .install/base/utils .install/base/settings .install/modules/data.atmosphere .install/modules/data.land
+$(call depends,models/cable): | .install/base/logger .install/base/utils
+$(call depends,models/clm45): | .install/base/logger .install/base/utils
+$(call depends,models/dalec): | .install/base/logger .install/base/remote .install/base/utils
 $(call depends,models/dvmdostem): | .install/base/utils
-$(call depends,models/ed): | .install/base/utils .install/base/remote .install/base/settings
-$(call depends,models/fates): | .install/base/utils .install/base/remote
-$(call depends,models/gday): | .install/base/utils .install/base/remote
-$(call depends,models/jules): | .install/base/utils .install/base/remote
-$(call depends,models/linkages): | .install/base/utils .install/base/remote
-$(call depends,models/lpjguess): | .install/base/utils .install/base/remote
-$(call depends,models/maat): | .install/base/utils .install/base/remote .install/base/settings
-$(call depends,models/maespa): | .install/base/utils .install/base/remote .install/modules/data.atmosphere
-$(call depends,models/preles): | .install/base/utils .install/modules/data.atmosphere
-$(call depends,models/sipnet): | .install/base/utils .install/base/remote .install/modules/data.atmosphere
-$(call depends,models/template): | .install/base/utils
-
+$(call depends,models/ed.Rcheck/PEcAn.ED2): | .install/base/utils
+$(call depends,models/ed): | .install/base/utils .install/base/logger .install/base/remote .install/base/settings
+$(call depends,models/fates): | .install/base/utils .install/base/logger .install/base/remote
+$(call depends,models/gday): | .install/base/utils .install/base/logger .install/base/remote
+$(call depends,models/jules): | .install/base/utils .install/base/logger .install/base/remote
+$(call depends,models/linkages): | .install/base/utils .install/base/logger .install/base/remote
+$(call depends,models/lpjguess): | .install/base/utils .install/base/logger .install/base/remote
+$(call depends,models/maat): | .install/base/logger .install/base/remote .install/base/settings .install/base/utils
+$(call depends,models/maespa): | .install/modules/data.atmosphere .install/base/logger .install/base/remote .install/base/utils
+$(call depends,models/preles): | .install/base/utils .install/base/logger .install/modules/data.atmosphere .install/base/utils
+$(call depends,models/sipnet): | .install/modules/data.atmosphere .install/base/logger .install/base/remote .install/base/utils
+$(call depends,models/template): | .install/base/logger .install/base/utils
```

Comment:
I think we decided we were going to skip installing the `models/template` package.
```suggestion
```

Reply:
skipped for now